### ISSUE TITLE
A11y : Cannot use 'Add Alertmanager' button via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46502]
**Steps to reproduce**:

In External Alertmanagers section. The '[Add Alertmanager' button](https://github.com/grafana/grafana/blob/main/public/app/features/alerting/unified/components/admin/ExternalAlertmanagers.tsx#L129-L142) is not clickable when no external alertmanagers have been added. elements inside the modal were focusable. Also, once an external alertmanager IS added, the button is keyboard clickable

**Actual Result**:

degraded experience for keyboard-only navigation

**Expected Result**

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by konrad147** [comment]
Hey Matt,
I cannot reproduce the issue. Can you provide more details (Grafana version, browser) or confirm that it works as expected?

**Comment by konrad147** [comment]
Closing because it works as expected. If you notice it's not working please provide more details and feel free to reopen.
